app/services/step_by_step_filter.rb

```diff
@@ -32,4 +32,8 @@ def filter_by_title_or_url
   def slug_from_title_or_url
     title_or_url.split('/').last
   end
+
+  def ordered_results(results)
+    results.sort_by(&:title)
```

Comment:
I wonder if it would be cleaner to call _this_ `results`, and to rename the method on line 9 to `unordered_results`? That way we wouldn't have to make all those `ordered_results` calls, and this method would simply be:

```ruby
def results
  unordered_results.sort_by(&:title)
end

def unordered_results
  if status.present? && title_or_url.present?
    filter_by_status & filter_by_title_or_url
  elsif ...etc
end
```

Reply:
I went with `filtered_results` rather than `unordered` because strictly speaking not all of the results are unordered at this point (`by_title` returns a list of all SBS ordered by title).

Updated and fixed up.